worlds/witness/data/WitnessLogic.txt

```diff
@@ -530,6 +530,7 @@ Town (Town) - Main Island - True - The Ocean - 0x0A054 - Town Maze Rooftop - 0x2
 158219 - 0x0A0C8 (Cargo Box Entry Panel) - True - Black/White Squares & Shapers
 Door - 0x0A0C9 (Cargo Box Entry) - 0x0A0C8
 158707 - 0x09F98 (Desert Laser Redirect Control) - True - True
+158904 - 0xFFD03 (Desert Laser Activated + Redirected) - 0x09F98 & 0x012FB - True
```

Comment:
Are these supposed to be 159804? Also should Thundercloud EP be changed to use this now?